### PR TITLE
⬆️ doctrine_migrations

### PR DESCRIPTION
**Pull request by damienmillet**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@
 /phpunit.xml
 .phpunit.result.cache
 ###< phpunit/phpunit ###
+
+composer.lock
```
Comment (XenoX) [inline .gitignore:26]: composer.lock must be push :)

File: config/packages/doctrine_migrations.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+doctrine_migrations:
```
Comment (XenoX) [inline config/packages/doctrine_migrations.yaml:1]: Are you sure this file is required ? I launch the migrations yesterday without this configuration file and it's worked

Comment (damienmillet) [reply config/packages/doctrine_migrations.yaml:1]: isn't needed for a migration (`d:m:m`) but needed for create the migration file (`make:migration`)

File: .gitignore (hunk 1 above)
Comment (damienmillet) [reply .gitignore:26]: it's a static file of installed app, when u deploy a new app u can hav lot of error like about the php version if different

File: config/packages/doctrine_migrations.yaml (hunk 2 above)
Comment (damienmillet) [reply config/packages/doctrine_migrations.yaml:1]: but u can remove the debug line x)

File: .gitignore (hunk 1 above)
Comment (XenoX) [reply .gitignore:26]: https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control

**Pull request closed without merging**